content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,294 @@
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in DVC Studio (other artifact types will not). The rest of the parameters are
+descriptive and optional and will also show up in DVC Studio.
+
+<details id="push-click-to-see-how-artifacts-are-registered">
+
+#### 💡 Expand to see how adding models to the registry works under the hood
+
+When we call the `log_artifact()` method, DVC takes all the information we
+provide in the call and edits the `dvc.yaml` file which will now contain the
+following lines:
+
+```yaml
+artifacts:
+  pool-segmentation:
+    path: models/model.pkl
+    type: model
+    desc:
+      This is a Computer Vision (CV) model that's segmenting out swimming pools
+      from satellite images.
+    labels:
+      - cv
+      - segmentation
+      - satellite-images
+```
+
+When you push the resulting file to your Git remote it is parsed by DVC Studio
+and it then shows all model artifacts from your `dvc.yaml` files in the model
+registry.
+
+</details>
+
+Now we just need to run the Python script which includes this code to cache and
+register the model. If you are
+[following](/docs/start/model-management/model-registry#follow-along-instructions)
+our example repository then this has already been done and we can continue to
+the next section.
+
+If you are building your own repository, you will need to run the script and
+push the result to your Git remote (GitHub) yourself. This is covered in detail
+in our
+[Experiment Management guide](/doc/start/experiments/experiment-pipelines).
+
+For a general introduction of how DVC stores and shares data, see our
+[Get Started guide on Data Versioning](/doc/start/data-management/data-versioning).
+
+<details id="push-click-to-see-other-ways-to-add-models">
+
+#### 💡 Expand to see other ways to add models
+
+The other two options are to use the DVC Studio's graphical user interface to
+add models interactively or to manually edit `dvc.yaml` files to add information
+about model artifacts. To get more details on the ways to add models have a look
+at the
+[Model registry documentation](/doc/studio/user-guide/model-registry/add-a-model).
+
+</details>
+
+## DVC Model registry overview
+
+In this guide, we will be using [DVC Studio](https://studio.iterative.ai) to
+manage our model registry. DVC Studio enables you to see models across all
+projects, manage their lifecycle, and download them with only a token. You can
+find out more about it [here](/doc/studio).
+
+From the Models tab in DVC Studio we will have an overview of all models, latest
+model versions as well stages each of the model versions is assigned to. We can
+get more details for each model by clicking on the model name.
+
+You can check out our
+[example model](https://studio.iterative.ai/team/Iterative/models/b3P4bcYIrGYdzyjqzsf9Xw==/pool-segmentation/v1.0.0)
```

Comment:
I'm getting a 404 for this link. I think we can drop this if we can't find a way to provide a stable link.

Reply:
I changed the link to something which does not exist yet, since it depends on merging [this](https://github.com/iterative/example-repos-dev/pull/271). I think the link should work then.